docs/api.md

```diff
@@ -2043,6 +2044,9 @@ page.on('requestfailed', request => {
 });
 ```
 
+#### request.frame()
+- returns: <?[Frame]> A matching [Frame] object, or `null` for offline mode or navigating to error pages.
```

Comment:
I wonder why this returns `null` for offline mode, this shouldn't be the case. Maybe that's because the request events are send *before* the frames are reported to puppeteer?

Reply:
I'll investigate deeper on this. I'll ask your help if it's difficult for me :)